gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/proc/ResumeDagProcTest.java

```diff
@@ -84,16 +84,19 @@ public void tearDown() throws Exception {
   @Test
   public void resumeDag() throws IOException, URISyntaxException {
     long flowExecutionId = System.currentTimeMillis();
```

Comment:
is there any reason NOT to just use a constant here?

Reply:
no reason